packages/core/schema/lavamoat-policy.v0-0-1.schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "LavaMoat Policy Schema",
+  "$id": "lavamoat-policy.v0-0-1",
+  "type": "object",
+  "description": "Schema for LavaMoat policy files",
+  "properties": {
+    "resources": {
+      "description": "Describe the resources available to your application and direct dependencies",
+      "title": "Resources",
+      "type": "object",
+      "$comment": "This one or more valid npm package names delimited by a \">\" character",
+      "patternProperties": {
+        "^(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*(>(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*)*$": {
+          "$ref": "#/$defs/resourcePolicy"
+        }
+      },
+      "additionalProperties": false
+    },
+    "resolutions": {
+      "type": "object",
+      "title": "Resolutions",
+      "description": "Custom run-time module resolutions by direct dependency",
+      "patternProperties": {
+        "^(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*(>(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*)*$": {
+          "type": "object",
+          "patternProperties": {
+            "^(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*(>(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*)*$": {
+              "type": "string",
+              "pattern": "^(\\.{1,2})(/(?=[^/\\0])[^/\\0]+)*/?$",
+              "minLength": 1,
+              "description": "Relative POSIX path to module from package root"
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "additionalProperties": false,
+  "anyOf": [{ "required": ["resources"] }, { "required": ["resolutions"] }],
+  "$defs": {
+    "resourcePolicy": {
+      "type": "object",
+      "title": "Resource Policy",
+      "properties": {
+        "globals": {
+          "title": "Globals",
+          "description": "Globals (including properties using dot notation) accessible to the module; `true` to allow and `false` to deny",
```

Comment:
(must) Also, `"write"` for write access.

The boolean/string mixup is deliberate, so that "write" really sticks out for the human reading this

Reply:
I also see `read` mentioned, but this may be legacy.